imap/pop3d.c

```diff
@@ -603,6 +603,8 @@ void shut_down(int code)
 
     in_shutdown = 1;
 
+    libcyrus_run_delayed();
```

Comment:
Some of these shut_down functions set `in_shutdown=1` before calling libcyrus_run_delayed(), others do it in the other order.

Might not be significant but it smells funny?

Reply:
Actually, we should always call it after setting in_shutdown, if the work hasn't been explicitly done already - because in_shutdown gets called by